drivers/sensor/sht3xd/sht3xd_trigger.c

```diff
@@ -195,7 +195,8 @@ int sht3xd_init_interrupt(struct device *dev)
 	}
 
 	gpio_pin_configure(drv_data->gpio, DT_SHT3XD_GPIO_ALERT_PIN,
-			   GPIO_DIR_IN | GPIO_INT | GPIO_INT_LEVEL |
+			   GPIO_DIR_IN | GPIO_INT |
+			   GPIO_INT_EDGE | GPIO_INT_DOUBLE_EDGE |
```

Comment:
This change should be in the previous commit

Reply:
I'd really rather not.  The sole purpose of the previous commit is to change the spelling of GPIO resource identifiers.  This changes the way the signal is generated, is required because of how the example works, and is essentially a behavior/functional change.  In practice I don't think the level interrupt could ever have worked because the application can't clear it, but nonetheless this change should either be in its own commit documenting why it was made (which I'm willing to do) or as part of the commit that motivated it (where it's documented in the commit comment).

Reply:
Agree, my mistake. I see a lot of pull requests that add a new sensor driver in one commit, then add a sample for that sensor in the next commit.

Let's make this change in its own commit.

Reply:
done